apis/iam/v1alpha1/referencers.go

```diff
@@ -20,12 +20,35 @@ import (
 	"context"
 	"fmt"
 
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 	"github.com/crossplane/crossplane-runtime/pkg/reference"
 	"github.com/crossplane/crossplane-runtime/pkg/resource"
 	"github.com/pkg/errors"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ServiceAccountReferer defines a reference to a ServiceAccount either via its RRN,
+//   or via a v1alpha1.ServiceAccount object or via a selector. RRN is the
+//   relative resource name as defined by Google Cloud API design docs here:
+//   https://cloud.google.com/apis/design/resource_names#relative_resource_name
+//   An example value for the ServiceAccount field is as follows:
+//   projects/crossplane-playground/serviceAccounts/perfect-test-sa@crossplane-playground.iam.gserviceaccount.com
```

Comment:
```suggestion
// ServiceAccountReferer defines a reference to a ServiceAccount either via its RRN,
// or via a v1alpha1.ServiceAccount object or via a selector. RRN is the
// relative resource name as defined by Google Cloud API design docs here:
// https://cloud.google.com/apis/design/resource_names#relative_resource_name
// An example value for the ServiceAccount field is as follows:
// projects/crossplane-playground/serviceAccounts/perfect-test-sa@crossplane-playground.iam.gserviceaccount.com
```

Is the IDE at work here?